ManatiCR/vitaminau
# VIT-30: Add webform uso de instalaciones

# Task [VIT-30](https://manati.atlassian.net/browse/VIT-30?atlOrigin=eyJpIjoiZjNiNDJkYTQwNTVjNGUwNGJmZjM4MzRjYjdkNzMyZDMiLCJwIjoiaiJ9)
Creation of the "Uso de instalaciones" form.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url: 
  - http://vitaminau.docker/form/uso-de-instalaciones
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-28](https://manati.atlassian.net/browse/VIT-30?atlOrigin=eyJpIjoiZjNiNDJkYTQwNTVjNGUwNGJmZjM4MzRjYjdkNzMyZDMiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save buttom
- Once you press it your information should be saved and the form should redirect you to a thank you page


# Acceptance criteria
- Only authenticated users can access the form.
- The following fields are not required
  - Ficha técnica
  - Acompañamiento
